docs/modules/quickstarters/partials/secret-scanning-with-gitleaks.adoc

```diff
@@ -0,0 +1,13 @@
+== Secret scanning with gitleaks
+
+This repository has a pre-commit hook that runs gitleaks on every commit.
+To install the hook, run:
+
+```
+pip install pre-commit
```

Comment:
Does that mean I would need to install python+pip on my machine to use this?

Reply:
consider also there are other ways to install it locally -> https://pre-commit.com/#install

Reply:
consider also there are other ways to install it locally -> https://pre-commit.com/#install

Reply:
> consider also there are other ways to install it locally -> https://pre-commit.com/#install

Do you mean by using homebrew or conda? One of them would need to be installed first as well to install pre-commit. 

But yes, the question is answered for me, thx!